adb_graphics/default_specs.yml

```diff
@@ -55,6 +55,31 @@
 #      anchor needs to be applied before another section.
 #
 
+1hsm: # Helicity
+  sfc: &hlcy # 0-1 km Storm Relative Helicity
+    clevs: [25, 50, 100, 150, 200, 250, 300, 400, 500, 600, 700, 800]
+    cmap: gist_ncar
+    colors: hlcy_colors
+    layer: 0
+    ncl_name: HLCY_P0_2L103_GLC0
+    ticks: 0
+    title: 0-1 km Storm Relative Helicity
+    unit: m2/s2
+1ref: # Composite reflectivity
```

Comment:
Don't we have another Composity Reflectivity section? If yes, how does this differ?

Reply:
my mistake, forgot to change the label after cut-and-paste.  1ref is the reflectivity 1 km AGL.